mmv1/third_party/terraform/tests/resource_logging_metric_test.go

```diff
@@ -128,11 +168,48 @@ resource "google_logging_metric" "logging_metric" {
 `, suffix, filter)
 }
 
+func testAccLoggingMetric_loggingBucketBase(suffix string, filter string) string {
+	return fmt.Sprintf(`
+resource "google_logging_metric" "logging_metric" {
+  name        = "my-custom-metric-%s"
+  filter      = "%s"
+
+  metric_descriptor {
+  	metric_kind = "DELTA"
+    unit        = "1"
+    value_type  = "INT64"
+  }
+}
+`, suffix, filter)
+}
+
+func testAccLoggingMetric_loggingBucket(suffix string, filter string, project_id string) string {
+	return fmt.Sprintf(`
+resource "google_logging_project_bucket_config" "logging_bucket" {
+  location  = "global"
+  project   = "%s"
+  bucket_id = "_Default"
+}
+
+resource "google_logging_metric" "logging_metric" {
+  name        = "my-custom-metric-%s"
+  bucket_name = google_logging_project_bucket_config.logging_bucket.id
+  filter      = "%s"
+
+  metric_descriptor {
+  	metric_kind = "DELTA"
+    unit        = "1"
```

Comment:
```suggestion
    metric_kind = "DELTA"
    unit        = "1"
```
only nit: we use use spaces instead of tabs for resource config